fastcov.py

```diff
@@ -874,6 +875,10 @@ def getGcovCoverage(args):
 
     # Get list of gcda files to process
     coverage_files = findCoverageFiles(args.directory, args.coverage_files, args.use_gcno)
+    if not coverage_files:
```

Comment:
I think we might need to move this check further down, below the `if args.zerocounters` line, or else it breaks the regression tests